### PR TITLE
Update how-to-compliant-network.md

### PR DESCRIPTION
**Pull request by sergiopozoh**
hi Ken. We need to make an adjustment here. We have this text reviewed by CELA that may require still normalization (like capitalization, use of quotes, etc.).

The changes in the product are in deployment and will be available in the next 2-3 weeks. I can tell you the exact date maybe next week. But wanted to open this ticket now so you have time.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@kenwith 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/global-secure-access/how-to-compliant-network.md [hunk 1]
```diff
@@ -80,11 +80,16 @@ The compliant network Conditional Access policy can be used to protect your Micr
 1. Select the **Create** button to create to enable your policy.
 
 > [!NOTE]
-> You can use Global Secure Access traffic profiles along with a Conditional Access policy requiring a compliant network for **All cloud apps**. There's no exclusion required when setting up a policy using the **All Compliant Network locations** location and **All cloud apps**.
-> 
-> Authentication to Global Secure Access traffic profiles are automatically excluded from Conditional Access enforcement when a compliant network is required. This exclusion enables the Global Secure Access client to access required resources to start and authenticate the user.
+> You can use Global Secure Access along with a Conditional Access policies requiring a Compliant Network for All Resources.
```
Comment (kenwith) [inline docs/global-secure-access/how-to-compliant-network.md:83]:
```suggestion
> Use Global Secure Access along with Conditional Access policies that require a Compliant Network for All Resources.
```

File: docs/global-secure-access/how-to-compliant-network.md [hunk 2]
```diff
@@ -80,11 +80,16 @@ The compliant network Conditional Access policy can be used to protect your Micr
...
+> Global Secure Access resources are automatically excluded from the Conditional Access policy when Compliant Network is enabled in the policy. There's no explicit resource exclusion required. These automatic exclusions are needed to ensure that the Global Secure Access client is not blocked from accessing the required resources Global Secure Access needs:
```
Comment (kenwith) [inline docs/global-secure-access/how-to-compliant-network.md:84]:
```suggestion
> Global Secure Access resources are automatically excluded from the Conditional Access policy when *Compliant Network* is enabled in the policy. There's no explicit resource exclusion required. You need these automatic exclusions to ensure that the Global Secure Access client is not blocked from accessing the required Global Secure Access resources. The resources Global Secure Access needs are: 
```

**Comment by kenwith** [comment]
#please-close

https://github.com/MicrosoftDocs/entra-docs-pr/pull/5770

**Pull request closed without merging**